tests/parametric/test_tracer.py

```diff
@@ -26,7 +26,7 @@ def test_tracer_span_top_level_attributes(self, test_agent: _TestAgentAPI, test_
                 "operation", service="my-webserver", resource="/endpoint", typestr="web"
             ) as parent:
                 parent.set_metric("number", 10)
-                with test_library.start_span("operation.child", parent_id=parent.span_id) as child:
+                with test_library.start_span("operation.child", parent_id=str(parent.span_id)) as child:
```

Comment:
Parent_id being a string comes from the fact it's passed as a string in the interface. But it's an implementaiton detail (I don't even know why). 

I think we should keep it an `int` on all our methods, becasue it's an `int`. And only tranfsom it to a `str` when serialized in `APMLibraryClient.trace_start_span`. It will also makes this PR smaller 😄  

WDYT ?